Core/StartupArgManager.cs

```diff
@@ -37,5 +39,32 @@ public static string[] MergeStartupArgs(IEnumerable<string> cmdArgs)
                 return startupArgs.ToArray();
             }
         }
+
+        public static void MigrateArgsFile()
```

Comment:
Added. Initially, I based my code off steam's `steamapps` folder. Since it usually doesn't require admin perms to install to. But after realizing how dumb that was, I wanted to implement my [admin detection script](https://gist.github.com/ALEXWARELLC/a46235adb80a19b840c77b0f808c6c03) but just decided that a try block would be better.